src/test/groovy/hudson/plugins/gradle/GradlePluginIntegrationTest.groovy

```diff
@@ -271,6 +272,11 @@ task hello << { println 'Hello' }"""))
         installation.name == 'myGradle'
         installation.home == '/tmp/foo'
 
+        def envVars = new EnvVars()
+        installation.buildEnvVars(envVars)
+        assert envVars.containsKey("PATH+GRADLE")
+        assert envVars.get("PATH+GRADLE") == installation.home + "/bin"
```

Comment:
Since the stated use case is for Declarative, perhaps you should have that as a test dep and demonstrate that

```groovy
tools {…}
steps {
  echo "PATH=$PATH"
}
```

works.